### ISSUE TITLE
1.9.0 release tracking

### ISSUE THREAD

**Issue by chrisohaver** [miekg/coredns#5144]
Proposing we release 1.9.0 now, which has some PRs queued up for it that change behavior in a backward incompatible way (although no configuration backward incompatibilities that would require Corefile updates)

* #5019 (issue: #4984)
* #5109

I don't think there are any pressing reasons to release a 1.8.8.

**Comment by yongtang** [comment]
Created a PR #5145 to add initial release note.

I am also thinking about change to use GitHub Actions to build and push container to dockerhub. This can be done whenever a release tag is created and triggered automatically. We had some issues in the past with docker. I will see if I can get this part in before 1.9.0.

**Comment by yongtang** [comment]
Created a PR #5146 for Github Actions to do docker release.

**Comment by yongtang** [comment]
Let's get 1.9.0 out first, as all have been covered.

**Comment by chrisohaver** [comment]
Has version been updated?

**Comment by yongtang** [comment]
Yes https://github.com/coredns/coredns/pull/5145/commits/b88a5e1681ee0591052c0b2e397add4274c54e65

**Comment by yongtang** [comment]
https://github.com/coredns/coredns/pull/5145/files

**Comment by chrisohaver** [comment]
@yongtang, if you don't have time to troubleshoot the new docker release action workflow failures.
Maybe just release the old way (via comment).

**Comment by yongtang** [comment]
The first error is due to rate limiting of GitHub API I believe so some download will be rate limited.

**Comment by yongtang** [comment]
The second time error seems to be due to the dockerhub credential.

**Comment by yongtang** [comment]
The 1.8.7's docker build failure might be also coming from rate limiting:
https://github.com/coredns/coredns/issues/4996#issuecomment-1013158130

So the response is not an tar.gz archive but HTTP rate limit I guess.

**Comment by chrisohaver** [comment]
> The 1.8.7's docker build failure might be also coming from rate limiting:
https://github.com/coredns/coredns/issues/4996#issuecomment-1013158130

I think that 1.8.7 failure was due to the release artifacts not being completely uploaded yet. i.e. it downloaded an incomplete file because it was still in the process of uploading. IIRC, I ran `/docker 1.8.7` as soon as I saw notifications that the _release_ was created.  However, this was before the release _script_ finished and wrote its status to the issue.
